src/plugins/polar/README.md

```diff
@@ -0,0 +1,96 @@
+Polar Coordinate Positioning Plugin
+===================================
+
+Preface
+-------
+This plugin includes and enhances functionality from the rel plugin.
+Please only use one of them.
+
+Relative Positioning (Cartesian)
+--------------------------------
+The plugin uses the following attributes for a step:
+	data-rel-ref
+	data-rel-x
+	data-rel-y
+	data-rel-z
+
+If the attribute "data-rel-ref" is not given, relative coordinates will be measured
+from the position of the previous frame. If added, the step with the given id determines
+the origin. This allows to group steps to a topic without the risk of scattering them when 
+one of them is removed.
+In addition to plain numbers, which are pixel values, it is also possible to
+define relative positions as a multiple of normal step height and width, using
+a unit of "h" or "w", respectively, appended to the number.
+
+Example:
+	<div id="relstep" class="step" 
+		data-rel-ref="main_step" data-rel-x="1w" data-rel-y="0.5h" data-rel-z="0">
+			
+Relative coordinates not given are inherited from the previous step, so make sure
+to set them to zero if that behaviour is not desired.
+Absolute values always override relative values!
+
+Polar Positioning
+-----------------
+
+To place steps e.g. in a circle around a central thought, polar positioning is introduced here.
+It uses the following attributes for a step:
+	data-rel-ref
+	data-rho
+	data-phi
+	data-rel-rho
+	data-rel-phi
+
```

Comment:
Why do you use rho and phi for polar coordinates? As rho is a greek letter it is associated most likely with an angle not a length. I think most rho and phi are common for spherical coordinates, as you have one more dimension. 
I would suggest `-distance` and `-phi` for polar coordinates and `-distance`, `-rho` and `-phi` or `-primary-angle` and `-secondary-angle` for spherical coordinates.
Have a look at https://en.wikipedia.org/wiki/Spherical_coordinate_system and https://en.wikipedia.org/wiki/Spherical_coordinate_system.

Reply:
Forces of habit. I learned to use rho, phi and theta forty years ago and still do ;-)

Reply:
BTW: x, y and z are also not 'quite' standard, here they are left-handed to comply with screen definitions...